src/software/pipeline/main_panoramaInit.cpp

```diff
@@ -216,6 +216,8 @@ class CircleDetector {
     double level_centerx = _center_x / pow(2.0, level);
     double level_centery = _center_y / pow(2.0, level);
     double level_radius = _radius / pow(2.0, level);
+    int min_radius = gradients.Width() / 2;
```

Comment:
To be independant from the image orientation:
```suggestion
    int min_radius = std::min(gradients.Width(), gradients.Height()) / 2;
```

Reply:
No ! Gradients is a polar image !